src/Analysis/Engine/Impl/Values/BuiltinNamespace.cs

```diff
@@ -101,7 +101,7 @@ class BuiltinNamespace<TMemberContainer> : AnalysisValue where TMemberContainer
 
         public virtual ILocatedMember GetLocatedMember() => null;
 
-        public override IEnumerable<ILocationInfo> Locations => GetLocatedMember()?.Locations.MaybeEnumerate();
+        public override IEnumerable<ILocationInfo> Locations => (GetLocatedMember()?.Locations).MaybeEnumerate();
```

Comment:
You probably can avoid `MaybeEnumerate` completely with `GetLocatedMember()?.Locations ?? Enumerable.Empty<>()`

Reply:
Yes, absolutely, I was just making the minimal change. I'll probably do that.